README.md

```diff
@@ -59,3 +59,37 @@ $ npm run dev
   proxy_pass http://wallet.v.systems:9922 
 ```
 to a particular address
+
+### Deploy using [zappa](https://github.com/Miserlou/Zappa)
+
+#### Installation
+
+
+```
+npm run build
```

Comment:
```npm run build``` is not an installation step

Reply:
Maybe change to Before-deploy and add npm i